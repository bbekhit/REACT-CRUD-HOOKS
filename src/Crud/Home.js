import React from "react";
import CrudLists from "./CrudLists";
import CrudForm from "./CrudForm";
import CrudAdd from "./CrudAdd";

export const Home = () => {
  return (
    <div>
      <CrudAdd />
      <CrudLists />
    </div>
  );
};
