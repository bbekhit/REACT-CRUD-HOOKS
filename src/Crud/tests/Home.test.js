import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import { Home } from "../Home";

test("should render Landing page correctly", () => {
  const wrapper = shallow(<Home />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
