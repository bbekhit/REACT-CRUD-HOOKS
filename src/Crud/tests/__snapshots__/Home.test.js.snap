// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Landing page correctly 1`] = `
<div>
  <Connect(CrudAdd) />
  <Connect(CrudLists) />
</div>
`;
